CHANGELOG.md

```diff
@@ -10,9 +10,11 @@ All notable changes to this project will be documented in this file.
 
 - Extended the `LdapAuthenticationProvider` with functionality to build add Volumes and Mounts to PodBuilder and ContainerBuilder ([#535]).
 - Extended the `PodBuilder` with `add_volume_with_empty_dir` utility function ([#536]).
+- Added method to create log4j2 config properties to product logging ([#540]).
```

Comment:
This entry belongs to the [Unreleased] section.

Reply:
Oh good find! https://github.com/stackabletech/operator-rs/pull/540/commits/001176e0abfd21514f29c16116ab3c7949c06c91